docs/source/reproducible.rst

```diff
@@ -104,3 +104,40 @@ configuration file and use it the script like so:
         # Set the seed for numpy
         np.random.seed(config.seed)
         # application-specific seed setting
+
+Instantiating objects
+---------------------
+
+Another cool feature of Hydra is object instantiating. Imagine you want to try different 
```

Comment:
could you put a link on the words "object instantiating" to the Hydra documentation? (that you showed me in the workshop)